### ISSUE TITLE
Snoozing is only available in my main inbox folder

### ISSUE THREAD

**Issue by Fr3llD** [nylas/N1#1779]
I am not sure this is by design or if it's a bug or simply not implemented yet, i'm just reporting the issue: Snoozing seems to be possible only on the main inbox emailfolder. I have several mailbox folders and the button on the rightcorner to snooze is only visible if i go to my main inboxfolder. 

- I checked  on;
https://support.nylas.com/hc/en-us/sections/203638587-N1_
https://github.com/nylas/N1/issues?q=is%3Aissue. 
but no hits, so im creating an issue.
- I am on Ubuntu 15.10
- I am using the latest N1 version delivered with Ubuntu 15.10 (system uptodate and patched)
- I am using Exchange 2016. 
- The issue is reproducable on my system.

**Comment by bengotow** [comment]
Hey! Thanks for reporting this—this is currently intentional, but we could expand support for snoozing from other places in the future. Our team thought that snoozing something from other places, especially Gmail labels, could be pretty confusing.

**Comment by Fr3llD** [comment]
Ok! Thanks for the reply.

If its by design, you can close this issue if you want. But if possible i would like to make it a feature request, especially for exchange users. This is a valuable addon for business users i think.

Ive been using emailclients for 20 years and this is the first time ive seen innovation in email, so keep up the good work!

Met vriendelijke groet,
Regards,
Dennis.

Verzonden vanaf een mobiele telefoon.

On 18 Mar 2016, at 01:23, Ben Gotow <notifications@github.com<mailto:notifications@github.com>> wrote:


Hey! Thanks for reporting this-this is currently intentional, but we could expand support for snoozing from other places in the future. Our team thought that snoozing something from other places, especially Gmail labels, could be pretty confusing.

-

**Comment by jstejada** [comment]
going to keep this open and change the name. thanks!

**Comment by osakkers** [comment]
Yeah - here's an issue I just had: it's just past midnight and I snoozed a message. I looked in the snoozed folder and it's snoozed not for tomorrow (i.e. 8 hours from now) but rather for the day after (32 hours from now). In the snoozed folder, there seems to be no way to adjust the snoozed email's setting, which is pretty frustrating. It would be good a) if snoozing it until for "tomorrow" wouldn't skip the day, even if it is after midnight and b) there was a way to change the snooze setting from the snooze folder. Also, tangentially related, a folder called "done" (as in Inbox) or something like that would be helpful for finding recently archived messages. Currently, it seems only search works for that :(

**Comment by stefaniwhylie** [comment]
I sometimes send emails, but want them to snooze (or pop up in my inbox) if someone hasn't responded.  So, it's sort of my reminder to follow-up with them.  So, snoozing from sent items, or somewhere other than inbox would work well.  To do this currently, I've been CC'ing or BCCing myself, which is not the best.

**Comment by jackiehluo** [comment]
Another request: https://github.com/nylas/N1/issues/2781

**Comment by stefaniwhylie** [comment]
Just wanted to check-in to see if there are any plans to implement this feature soon.  Snooze from anywhere other than the inbox would be great.  Snoozing from send is great, but even just snoozing from search (inbox items) would be great. 

Right now I get a ton of email daily, and sometimes I do searches like "after:2016/10/3 label:inbox" just to get the emails I've received within the last day or so.  It would be great to be able to snooze these, but even that doesn't work.  As I understand it, you can't snooze from anywhere other than the inbox currently, but I'm thinking that I should at least be able to snooze from search, especially if the item I'm trying to snooze is located in the inbox. 

My workaround currently is to go through all the other items, leave the items I want to snooze, then go to the inbox, without searching, and snooze any that are leftover.  Still pretty painful since I haven't been able to see inbox zero in ages. 

Thanks, guys!
